examples/summarization/finetune.py

```diff
@@ -85,7 +85,7 @@ def __init__(self, hparams, **kwargs):
         if self.hparams.freeze_encoder:
             freeze_params(self.model.model.encoder)  # TODO: this will break for t5
         self.hparams.git_sha = get_git_info()["repo_sha"]
```

Comment:
if you want, we can add this native to PL so you don't have to do it

Reply:
Yeah that would be cool!